### PR TITLE
A few additions

### PR DESCRIPTION
**Pull request by isaacminogue**
What a great list! Thanks for compiling it. Feel free to take or leave any of these, or reword them if you agree, but just don't think they fit the convention you've established. They're just some common ones I go to.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +27,9 @@ The [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) spec is deliberately not dup
 - Font weights below 400 should not be used
 - Medium sized headings generally look best with a font weight between 500-600
 - Adjust values fluidly by using CSS [`clamp()`](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp), e.g. `clamp(48px, 5vw, 72px)` for the `font-size` of a heading
+- Tabular figures should be used in lists where making comparisons between numbers are important, or layout shifts are undesirable, like in timers
```
Comment (raunofreiberg) [inline README.md]: Could you elaborate on what "making comparisons between numbers" means? I understand how this helps with not having layout shift but that part is fuzzy for me. Perhaps we can also mention `font-variant-numeric: tabular-nums`?

File: README.md [hunk 2]
```diff
@@ -71,6 +78,8 @@ The [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) spec is deliberately not dup
 - Images should always be rendered with `<img>` for screen readers and ease of copying from the right click menu
 - Illustrations built with HTML should have an explicit `aria-label` instead of announcing the raw DOM tree to people using screen readers
 - Gradient text should unset the gradient on `::selection` state
+- When using submenus, make sure projected mouse trajectories over non interactive elements don’t close the menu if moved above a certain speed threshold
```
Comment (raunofreiberg) [inline README.md]: Does this reference [this](https://twitter.com/JohnPhamous/status/1657083267299028992) problem?

File: README.md [hunk 3]
```diff
@@ -60,6 +66,7 @@ The [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) spec is deliberately not dup
 
 ## Accessibility
 
+- Set `meta` tag in `html` to `content="width=device-width, initial-scale=1.0”`
```
Comment (raunofreiberg) [inline README.md]: Don't most frameworks do this out of the box, and is this even required for modern browsers? Haven't set this myself for ages!

**Review by raunofreiberg — COMMENTED** [review]
This is great stuff, a few comments!

File: README.md (hunk 1 above)
Comment (raunofreiberg) [reply README.md]: Do you mean visually diffing between two numbers? I can see how that would make sense yeah! Maybe we can word it better...

Comment (isaacminogue) [reply README.md]: Yeah, exactly.  Ie, it's easier to see the true size of the values. Feel free to remove that part though if it's too vague.

Friday 02 June

File: README.md (hunk 2 above)
Comment (isaacminogue) [reply README.md]: YES. I just couldn't find an example.

File: README.md (hunk 3 above)
Comment (isaacminogue) [reply README.md]: Fair call. It's like a html 101 thing eh. Too obvious?

File: README.md (hunk 2 above)
Comment (isaacminogue) [reply README.md]: Maybe reword to something along these lines: 

> When using nested menus, use a "prediction cone" to prevent the pointer from accidentally closing the menu when moving across other elements.

Comment (raunofreiberg) [reply README.md]: I like that more!

File: README.md (hunk 3 above)
Comment (raunofreiberg) [reply README.md]: Yeah I think so

File: README.md (hunk 1 above)
Comment (raunofreiberg) [reply README.md]: I like this but would mention `font-variant-numeric` :)

**Comment by isaacminogue** [comment]
Hey @raunofreiberg — just applied all the suggested changes.

**Review by raunofreiberg — APPROVED** [review]
Looks great, thank you ❤️
